doc/source/Parameters.rst

```diff
@@ -330,7 +330,7 @@ solar metallicity and scaled linearly with the metallicity of the gas.
 
 Valid range:
 
- - number density: -10 < log\ :sub:`10` (n\ :sub:`H` / cm\ :sup:`-3`) < 4
+ - number density: -10 < log\ :sub:`10` (n\ :sub:`H` / cm\ :sup:`3`) < 4
```

Comment:
The "-" sign should be kept there since n is units of cm^-3, similar to how temperature is expressed as log (T / K).